test/benchmarks/benchmark_adler32.cc

```diff
@@ -69,11 +69,14 @@ BENCHMARK_ADLER32(c, adler32_c, 1);
 
 #ifdef ARM_NEON_ADLER32
 BENCHMARK_ADLER32(neon, adler32_neon, arm_cpu_has_neon);
-#elif defined(POWER8_VSX_ADLER32)
-BENCHMARK_ADLER32(power8, adler32_power8, power_cpu_has_arch_2_07);
-#elif defined(PPC_VMX_ADLER32)
+#endif
+
+#if defined(PPC_VMX_ADLER32)
```

Comment:
Perhaps use `#ifdef` instead of `#if defined(..)` in both places.. ?

Reply:
Done.